web/src/components/MapView.tsx

```diff
@@ -94,6 +82,7 @@ const MapView = React.forwardRef((props: MapViewProps, ref: React.Ref<MapRef>):
     paint: {}
   }
   const [viewport, setViewport] = useState<MapViewViewport>(bboxViewport)
+  const [cursor, setCursor] = useState<string>('auto')
```

Comment:
I think it would be nice to have a more restrictive type here

Reply:
gonna be added in 902